timer.py

```diff
@@ -6,20 +6,27 @@ def __init__(self):
         # set the font and the starting ticks for calculating timer
         self.font = pygame.freetype.SysFont(None, 34) #34 font size
         self.font.origin=True
-        self.start_ticks =  pygame.time.get_ticks() 
+        self.start_ticks =  0
+        self.running = False
 
+    def start_timer(self):
+        if not self.running:
+            self.start_ticks =  pygame.time.get_ticks() 
+            self.running = True
 
-    def update_timer(self, clock, display):
+    def is_running(self):
+        return self.running
+
+    def update_timer(self, display):
         total_ticks = pygame.time.get_ticks()
 
-        seconds = TIMER_SECONDS - int((total_ticks - self.start_ticks) /1000) #divided by millisconds
+        seconds = TIMER_SECONDS - int((total_ticks - self.start_ticks) /1000) #divided by milliseconds
 
         #when the timer ends, may want to change this to switch pages instead of quit
         if seconds < 0:
+            self.running = False
             sys.exit()
 
         #display the timer
         out = '{seconds:02d}'.format(seconds=seconds)
-        self.font.render_to(display, (DISPLAY_SIZE[0] // 2.1, DISPLAY_SIZE[1] // 20), out, pygame.Color('White'))
-        pygame.display.flip()
-        clock.tick(60)
```

Comment:
Were lines 23 and 24 not needed? Or were they part of the reason it was running slow?

Reply:
No, lines 24 and 25 are what made it run slow.  Idk why git assumed I changed line 23....  I added line 25 back to the run file where it was initially.  Although, I did test the game with and without the clock tick, and there was no difference that I can see.